docs/pages/access-controls/compliance-frameworks/fedramp.mdx

```diff
@@ -42,9 +45,10 @@ $ sudo ./install
 
 ## Configuration
 
-### Teleport Auth Server
+### Teleport Auth Service
 
-Now, save the following configuration file as `/etc/teleport.yaml` on the auth server.
+Now, save the following configuration file as `/etc/teleport.yaml` on the Auth
```

Comment:
```suggestion
Now, save the following configuration file as `/etc/teleport.yaml` on the Teleport Auth
```
just because this makes it more like a proper name not because I prefer it 😄